### PR TITLE
Use maintained gziphandler

### PR DESCRIPTION
**Pull request by earl-warren**
- https://github.com/NYTimes/gziphandler doesn't seems to be maintained anymore and the code already includes
https://github.com/klauspost/compress which provides a maintained and faster gzip handler fork.
- Enables Jitter to prevent BREACH attacks, as this *seems* to be possible in the context of the server.

(cherry picked from commit 45a16f8c3c6f7d5e4aab8fdde6a621cf36e4801c)

### CODE REVIEW

File: routers/web/web.go [hunk 1]
```diff
@@ -49,17 +49,12 @@ import (
...
-	"github.com/NYTimes/gziphandler"
 	chi_middleware "github.com/go-chi/chi/v5/middleware"
 	"github.com/go-chi/cors"
+	"github.com/klauspost/compress/gzhttp"
 	"github.com/prometheus/client_golang/prometheus"
 )
 
-const (
-	// GzipMinSize represents min size to compress for the body size of response
-	GzipMinSize = 1400
```
Comment (lunny) [inline routers/web/web.go:60]: How about `GzipMinSize = gzhttp.DefaultMinSize`?

**Comment by lunny** [comment]
The library's License looks like some complicated.

**Comment by jolheiser** [comment]
> The library's License looks like some complicated.

It looks like the `gzhttp` package simply uses its own apache2 license vs the base module also including the Go authors.

**Comment by KN4CK3R** [comment]
The removed license should be equal to the added one because the new code is a fork of the replaced code.

**Review by delvh — APPROVED** [review]

**Pull request closed without merging**